bin/katello-certs-check

```diff
@@ -53,6 +53,27 @@ function success () {
     echo "[OK]"
 }
 
+function check-expiration () {
+  DATE=$(date -u +"%b %-d %R:%S %Y")
```

Comment:
Sorry, missed that.
I will submit that in the next commit.
Looking over some other methods of verification before I finalize.